src/Components/SignUp/CreateAnAccount/CreateAnAccount.jsx

```diff
@@ -151,10 +151,17 @@ const CreateAnAccount = () => {
       });
   };
 
+  const isFullNameValid = (fullname) => {
+    var format = /[ `!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?~]/;
```

Comment:
This is not a valid regular expression to check the full name

Reply:
This will not work for all the scenario. Please use something like `^[a-z]([-']?[a-z]+)*( [a-z]([-']?[a-z]+)*)+$ `

Reply:
@biswanathsahoo , close this one, created new pr with new changs